packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -27,77 +38,97 @@ import {
 export class AtomicQuerySummary implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
   public querySummary!: QuerySummary;
+  private unescapeStringOption = {interpolation: {escapeValue: false}};
 
   @BindStateToController('querySummary')
   @State()
   private querySummaryState!: QuerySummaryState;
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    noResults: () => this.bindings.i18n.t('noResults'),
+    noResultsFor: (query: string) =>
+      this.bindings.i18n.t('noResultsFor', {
+        ...this.unescapeStringOption,
+        query,
+      }),
+    showingResultsOf: (resultOfOptions: ResultOfOptions) =>
+      this.bindings.i18n.t('showingResultsOf', {
+        ...this.unescapeStringOption,
+        ...resultOfOptions,
```

Comment:
Instead of passing html strings and needing to specify the `unescapeStringOption` option,
wouldn't it be simpler to pass in the state values returned by the query summary controller?

Reply:
We need the highlights to be wrapped in tags, just having the state values won't allow us to do this without postprocessing the string